docs/whats-new/preview-3.md

```diff
@@ -0,0 +1,446 @@
+
+:::image type="content" source="media/preview-3/trace-duration.png" lightbox="media/preview-3/trace-duration.png" alt-text="Trace details page with new radial icon used to quickly differentiate trace lengths.":::
+
+### New resource details view
+
+The resources page grid was bursting at the seams with information. In preview 3 there's now a resource details view. The grid continues to display important information, and additional details, including all of a resources properties, are available in the details view.
+
+:::image type="content" source="media/preview-3/resource-details-view.png" lightbox="media/preview-3/resource-details-view.png" alt-text="Resource details view.":::
+
+### Telemetry improvements
+
+The dashboard telemetry pages show off a number of improvements:
+
+- **Metrics filter UI**: The metrics filter UI has been redesigned. It's now much faster to update filters when viewing a metrics chart.
+- **Histogram metrics count**: There is a new option to view the count of items recorded to a histogram metric.
+- **Span events**: OpenTelemetry spans support events. For example, when was the first data received during a database call. Events are available in span details view.
+
+## Component updates
+
+[.NET Aspire components](../fundamentals/components-overview.md) are a curated suite of NuGet packages specifically selected to facilitate the integration of cloud-native applications with prominent services and platforms. Each component furnishes essential cloud-native functionalities through either automatic provisioning or standardized configuration patterns. .NET Aspire components can be used without an orchestrator project, but they're designed to work best with the .NET Aspire app host.
+
+### Azure AI OpenAI component
+
+This component enables integration of the Azure AI OpenAI or OpenAI services from a .NET Aspire project. Assuming you already have an OpenAI service provisioned, you can use OpenAI from your projects.
+
+- [Aspire.Azure.AI.OpenAI](https://www.nuget.org/packages/Aspire.Azure.AI.OpenAI) NuGet 📦 package.
+
+```csharp
+using Azure.AI.OpenAI;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add service defaults and components.
+builder.AddServiceDefaults();
+builder.AddAzureOpenAI("OpenAI");
+
+var app = builder.Build();
+
+app.MapGet("/chat", static async (string request, OpenAIClient client) =>
+{
+    var options = new ChatCompletionsOptions(
+        "gpt-35-turbo", [new ChatRequestUserMessage(request)]);
+
+    var response = await client.GetChatCompletionsAsync(options);
+
+    return response.Value.Choices.FirstOrDefault()?.Message.Content
+        ?? "no response...";
+});
+
+app.MapDefaultEndpoints();
+
+app.Run();
+```
+
+For more information, see [GitHub: .NET Aspire.Azure.AI.OpenAI](https://github.com/dotnet/aspire/tree/main/src/Components/Aspire.Azure.AI.OpenAI).
```

Comment:
Does OpenAI need anything for the Hosting project?

Reply:
Yeah, see https://github.com/dotnet/aspire/blob/main/src/Aspire.Hosting.Azure/AzureOpenAIResource.cs